sdk/v2/core/healthcheck.go

```diff
@@ -0,0 +1,43 @@
+package core
+
+import (
+	"context"
+	"net/http"
+
+	rm "github.com/brigadecore/brigade/sdk/v2/internal/restmachinery"
+	"github.com/brigadecore/brigade/sdk/v2/restmachinery"
+)
+
+// HealthcheckClient is the specialized client for accessing healthchecks on
+// the Brigade API.
+type HealthcheckClient interface {
+	// Ping pings the API Server
+	Ping(ctx context.Context) error
+}
```

Comment:
A thought...

We should probably at some point also expose an endpoint that returns the server's version information.

I'm wondering if it might make sense to lump `Ping()` and `Version()` together in one client-- maybe one named "system" API client or some such?

To clarify-- not asking for `Version()` to be added to this PR. Just trying to plan ahead is all.

Reply:
Yea, I like this.  `SystemClient`?

Reply:
Yes... or even call it `APIClient` and stick it in the `system` package.

Reply:
`core` probably isn't the right place for it. `core` is just for projects, events, etc.